docs/PaymentButtons/README.md

```diff
@@ -44,20 +46,31 @@ class MyViewController: ViewController {
         return payPalButton
     }()
     
-    @objc func paymentButtonTapped() {
+    @objc func payPalButtonTapped() {
         // Insert your code here
     }
     
     override func viewDidLoad() {
         super.viewDidLoad()
         view.addSubview(payPalButton)
+
+        // example constraints to absolutely center the button with a 20 pt. horizontal margin
+        let safeArea = view.safeAreaLayoutGuide
+        NSLayoutConstraint.activate([
+            view.leadingAnchor.constraint(equalTo: safeArea.leadingAnchor, constant: 20)
+            view.trailingAnchor.constraint(equalTo: safeArea.trailingAnchor, constant: -20)
+            view.centerXAnchor.constraint(equalTo: safeArea.centerXAnchor)
+            view.centerYAnchor.constraint(equalTo: safeArea.centerYAnchor)
```

Comment:
nit: we shouldn't need a center X constraint if we have a leading and trailing constraint. Also, we may need a height constraint unless our SDK handles height of the button?

Reply:
Oh right yeah good call. I can remove the center X constraint. The base class `UIButton` should have an intrinsic height based on the font size, so we should be good there.